[Package]: rig

### Package description

RIG stands for Random Identity Generator. It is a free replacement for a shareware program out there called 'fake'. Basically its single purpose in life is to generate random, yet real-looking, personal data.

### Home page URL

http://rig.sourceforge.net/

### Source code URL

https://sourceforge.net/projects/rig/files/rig/1.11/rig-1.11.tar.gz/download

### Packaging policy acknowledgement

- [X] I have read and understand the [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

I tried to compile it but it failed. There's no traditional configure file etc. and i'm a beginner.

It's said in the README just "make install" but I got this : 
```
make install 
install -g 0 -m 755 -o 0 -s rig /usr/local/bin
install: cannot create regular file '/usr/local/bin': No such file or directory
make: *** [Makefile:14: install] Error 1
```

Anyway it's just for fun, not priority stuff but I thought it would be nice to have it !

--- a/packages/rig/Makefile.patch
+++ b/packages/rig/Makefile.patch
@@ -1,0 +1,17 @@
+--- a/Makefile
++++ b/Makefile
+@@ -1,11 +1,11 @@
+-PREFIX=/usr/local
++PREFIX=@TERMUX_PREFIX@
+ BINDIR=${PREFIX}/bin
+-MANDIR=${PREFIX}/man
++MANDIR=${PREFIX}/share/man
+ DATADIR=${PREFIX}/share/rig
+ 
+ all: rig rig.6
+ rig: rig.cc
+-	g++ -g rig.cc -o rig -Wall -DDATADIR="\"$(DATADIR)\""
++	$(CXX) $(CXXFLAGS) $(CPPFLAGS) rig.cc $(LDFLAGS) -o rig -Wall -DDATADIR="\"$(DATADIR)\""
+ 
+ rig.6: rig.6.in
+ 	sed s@DATADIR@"$(DATADIR)"@g < rig.6.in > rig.6

--- a/packages/rig/build.sh
+++ b/packages/rig/build.sh
@@ -1,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=http://rig.sourceforge.net/
+TERMUX_PKG_DESCRIPTION="A program that generates fake identities"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.11
+TERMUX_PKG_SRCURL=https://downloads.sourceforge.net/rig/rig/rig-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=00bfc970d5c038c1e68bc356c6aa6f9a12995914b7d4fda69897622cb5b77ab8
+TERMUX_PKG_DEPENDS="libc++"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_make_install() {
+	install -Dm700 -t $TERMUX_PREFIX/bin rig
+	install -Dm600 -t $TERMUX_PREFIX/share/man/man6 rig.6
+	install -Dm600 -t $TERMUX_PREFIX/share/rig data/*.idx
+}

